README.md

```diff
@@ -170,5 +200,7 @@ Write what an example ERB file (aka view) might look like to list all the warrio
 
 Write your code here (**NOTE: syntax highlighting doesn't work for ERB in markdown files, so ignore the colors!**):
 ```html
-<!-- code here -->
+
+  <% @warriors.each do |warrior| %>
+
```

Comment:
Looks like you're missing an `<% end %>` and the code in the each block to actually display each warriors name: `<%= warrior.name %>`